en/getting_started/installation.md

```diff
@@ -0,0 +1,75 @@
+# Installation
+
+This topic explains how to install/set up MAVSDK for all our supported programming languages/platforms.
+
+
+## C++
+
+This section show how to install the MAVSDK core library on various platforms.
+
+> **Note** For most supported platforms you can download and install pre-built binaries.
+  During development (and for some platforms - e.g. Windows), you must [build the library from source](../contributing/build.md).
+
+### Ubuntu
+
+Download the **.deb** file for your system from [MAVSDK releases](https://github.com/mavlink/MAVSDK/releases) and install it using `dpkg`:
+
+```
+sudo dpkg -i mavsdk_0.24.0_ubuntu18.04_amd64.deb
+```
+
+### macOS
+
+Use [brew](https://brew.sh/) to install the library:
+
+```
+brew install mavsdk
+```
+
+### Windows
+
+> **Note** No binaries for Windows are available yet.
+  For now you must [build the library from source](../contributing/build.md).
+
+### Fedora
```

Comment:
I would keep all Linux distributions together, so I'd just move Ubuntu down to here.